homeassistant/components/motionblinds_ble/entity.py

```diff
@@ -0,0 +1,50 @@
+"""Base entity for the Motionblinds BLE integration."""
+
+import logging
+
+from motionblindsble.device import MotionDevice
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_ADDRESS
+from homeassistant.helpers.device_registry import CONNECTION_BLUETOOTH, DeviceInfo
+from homeassistant.helpers.entity import Entity
+
+from .const import CONF_BLIND_TYPE, CONF_MAC_CODE, MANUFACTURER
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class MotionblindsBLEEntity(Entity):
+    """Base class for Motionblinds BLE entities."""
+
+    _attr_has_entity_name = True
+    _attr_should_poll = False
+
+    _device: MotionDevice
+    config_entry: ConfigEntry
+
+    def __init__(
+        self,
+        device: MotionDevice,
+        entry: ConfigEntry,
+        unique_id_suffix: str | None = None,
+    ) -> None:
+        """Initialize the entity."""
+        self._attr_unique_id: str = (
+            entry.data[CONF_ADDRESS]
+            if unique_id_suffix is None
+            else f"{entry.data[CONF_ADDRESS]}_{unique_id_suffix}"
+        )
+        self._device = device
+        self.config_entry = entry
+        self._attr_device_info = DeviceInfo(
```

Comment:
We should add an identifiers

Reply:
I was told to use `connections` here https://github.com/home-assistant/core/pull/109497#discussion_r1482222710

Reply:
Oh interesting, I don't have any BLE integration made so i am actually not sure